tools/seeds/seeds.go

```
@@ -0,0 +1,225 @@
+
+	return nil
+}
+
+// SeedPrices reads sample_prices_2020.csv and bulk-inserts adjusted_price rows.
+// Uses h.CsvPath for the CSV file location.
+func (h *Hammer) SeedPrices() error {
+	f, err := os.Open(h.CsvPath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `f.Close` is not checked

(errcheck)